tgui/packages/tgui/styles/components/Button.scss

```diff
@@ -137,18 +135,29 @@ $bg-map: colors.$bg-map !default;
   @include button-color($color-selected);
 }
 
-.Button--verticalAlign--top {
-  align-items: flex-start;
+.Button--flex--inline{
+  display: inline-flex;
+  flex-direction: column;
 }
-  
-.Button--verticalAlign--middle {
-  align-items: center;
+
+// Companion to fluid prop
+.Button--flex--block{
+  display: flex;
+  flex-direction: column;
+}
+
+.Button--verticalAlignContent--top {
+  justify-content: flex-start;
 }
   
-.Button--verticalAlign--bottom {
-  align-items: flex-end;
+.Button--verticalAlignContent--middle {
+  justify-content: center;
 }
   
-.Button__content--fluid {
-  flex: 1 0 auto;
+.Button--verticalAlignContent--bottom {
+  justify-content: flex-end;
+}
+
+.Button__content{
+  display:block;
```

Comment:
Switched over to flex-direction: column so this can take up the full width and let text-align work properly.

Reply:
So now without verticalAlignContent set this will be the only thing changing.